christfellowshipchurch/web-app-v2
# CFE Royal Palm Beach Set a Reminder Update

### About
This PR adds the new CFE RPB set a reminder video to the page.

### Test Instructions
1. Go to [/locations/iglesia-royal-palm-beach](https://web-app-v2-git-cfe-set-a-remind-f51aff-christ-fellowship-church.vercel.app/locations/iglesia-royal-palm-beach) in the testing link and above the `Establece un recordatorio para el próximo servicio` text you should see the video, compare with the [live](https://www.christfellowship.church/locations/iglesia-royal-palm-beach) page to see the difference. 

### Screenshots
<img width="1440" alt="Screenshot 2024-08-21 at 2 44 14 PM" src="https://github.com/user-attachments/assets/ca197ee7-f47b-4120-ae6b-aa00595f669b">

### Closes Tickets
[CFDP-3165](https://christfellowshipchurch.atlassian.net/browse/CFDP-3165)


[CFDP-3165]: https://christfellowshipchurch.atlassian.net/browse/CFDP-3165?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
